ruby/red-arrow/test/test-record-batch-raw-records.rb

```diff
@@ -0,0 +1,367 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class RecordBatchRawRecordsTest < Test::Unit::TestCase
+  sub_test_case("with basic arrays") do
+    def setup
+      @string_values = ["apple", "orange", "watermelon", nil, "タコ"]
+      @string_array = Arrow::StringArray.new(@string_values)
+
+      @uint32_values = [1, 2, 4, 8, 16]
```

Comment:
We should also use a value between `2 ** 16` and `2 ** 32 - 1`.

Reply:
fixed.